src/InputWithLabel.jsx

```diff
@@ -0,0 +1,27 @@
+import { useRef, useEffect } from 'react';
+
+
+function InputWithLabel({ children, value, onChange}) {
+    const inputRef = useRef(null);
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
+
+    return (
+        <>
+            <label htmlFor="todoTitle">{children}Title </label>
```

Comment:
Please delete the word 'Title'—it's a duplicate since you've already passed it as {children}.

Reply:
fixed